Streetcode/Streetcode.XIntegrationTest/ControllerTests/AdditionalContent/CoordinateControllerTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using FluentResults;
+using Streetcode.BLL.DTO.AdditionalContent.Coordinates;
+using Streetcode.BLL.DTO.AdditionalContent.Coordinates.Types;
+using Streetcode.XIntegrationTest.ControllerTests.Utils;
+using System.Net.Http.Json;
+using System.Text.Json;
+using Xunit;
+
+namespace Streetcode.XIntegrationTest.ControllerTests.AdditionalContent
+{
+    public class CoordinateControllerTests: BaseControllerTests, IClassFixture<CustomWebApplicationFactory<Program>>
+    {
+        public CoordinateControllerTests(CustomWebApplicationFactory<Program> factory):base(factory,"/api/Coordinate")
+        {
+        }
+
+        [Fact]
+        public async Task GetByStreetcodeId_SuccsessStatusCode()
+        {
+            int id = 1;
+            var response = await client.GetByIdAsync(id);
+
+            Assert.True(response.IsSuccessStatusCode);
```

Comment:
In addition to status code you should check if returned object is the one we expect